src/camera.cc

```diff
@@ -2,6 +2,7 @@
 #include "ray.h"
 #include "scene.h"
 #include "scene_object.h"
+#include "point_light.h"
```

Comment:
We need to refactor later, as it makes no sense to need point_light.h in the camera class